doc/integrator/install_application.rst

```diff
@@ -257,7 +257,13 @@ You can build and install the application with the command:
 
     ./docker-run make --makefile=<user>.mk build
 
-Create the application tables, and directly set the version (details later):
+This previous command will do many things like:
+
+  * adapt the application configuration to your environment,
+
+  * build the javascript and css resources into compressed files,
+
+Then create the application tables, and directly set the version (details later):
```

Comment:
I don't understand the "and directly set the version" part. I know this part was already there, before your PR. But maybe we can clarify it? If there are "details later", we should explicitly point to them.

Reply:
ping  @sbrunner could you explain this? :) I will update it this afternoon (or later).

Reply:
There is a table in <schema_name>_static named alembic which contains version of alembic.

Reply:
For me this a relay old comment and can be removed ...